src/modules/Dashboard/features/Applet/Overview/Overview.utils.tsx

```diff
@@ -0,0 +1,151 @@
+import { PropsOf } from '@emotion/react';
+import { IconButton } from '@mui/material';
+import { formatDistanceStrict } from 'date-fns';
+import { enUS, fr } from 'date-fns/locale';
+
+import i18n from 'i18n';
+import { GetAppletSubmissionsResponse } from 'api';
+import { ParticipantSnippet } from 'modules/Dashboard/components';
+import { QuickStats } from 'modules/Dashboard/features/Applet/Overview/QuickStats';
+import { StyledMaybeEmpty } from 'shared/styles/styledComponents/MaybeEmpty';
+import { StyledFlexAllCenter, StyledFlexTopCenter, theme, variables } from 'shared/styles';
+import { Svg } from 'shared/components';
+
+const locales = { en: enUS, fr };
+
+export function mapResponseToQuickStatProps(
+  { participantsCount, submissionsCount }: GetAppletSubmissionsResponse = {
+    participantsCount: 0,
+    submissionsCount: 0,
+    submissions: [],
+  },
+  commonProps: {
+    onPressAddParticipant?: () => void;
+  },
+): PropsOf<typeof QuickStats> {
+  return {
+    stats: [
+      {
+        label: i18n.t('appletOverview.statParticipants'),
+        icon: 'profile' as const,
+        value: (
+          <StyledFlexTopCenter component="span" sx={{ gap: 1.6 }}>
+            {(participantsCount ?? 0).toLocaleString(i18n.language)}
+
+            <IconButton onClick={commonProps.onPressAddParticipant}>
+              <Svg aria-label={i18n.t('addParticipant')} id="add" width={20} height={20} />
+            </IconButton>
```

Comment:
I've pushed some updates that should hopefully improve the look of things here. The stats UI has been tweaked to match the latest revisions, and while the [Overview screen in Figma](https://www.figma.com/design/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?node-id=6934-109894&t=UR48vWAwTRF282b0-4) still depicts four items, the [Stats component itself](https://www.figma.com/design/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?node-id=6934-109894&t=UR48vWAwTRF282b0-4) includes design variations for 1 or more.

I don't see any designs that include the "Add Participant" button at the moment, but I've included a variation similar to what was previously present, [shown here](https://private-user-images.githubusercontent.com/1670836/335371352-6b2379e9-3c08-4b0e-b67c-fe287ac54d9b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcxMDU4MTcsIm5iZiI6MTcxNzEwNTUxNywicGF0aCI6Ii8xNjcwODM2LzMzNTM3MTM1Mi02YjIzNzllOS0zYzA4LTRiMGUtYjY3Yy1mZTI4N2FjNTRkOWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MzBUMjE0NTE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmQ0MGU0ODI3NWNmNWMzNzZkZTI1NmQzNTQ0ZjgxMzkwODU3NDQ3ZjA2M2I1MWU3NGZjMzU2ZmUyNGQ0OWRmYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.8tD0qIikOCmUqZx9h58BljHnyI1AoZdXKjw615fgdtM) in the updated screenshots I added. Mostly from my own reservations about accessibility, I'm a bit reluctant to make this hidden until mouse-over, unless the designers explicitly request it.

Reply:
Changes look great! But looking at the screens now, it appears the screen that displays an "Add Participant" button has been removed altogether. Maybe it was decided that adding participants from this screen no longer makes sense? I personally like it, but I think it might also be good to confirm with design, and if so, make sure to mention this in JIRA to set QA expectations when they compare against Figma.

Reply:
@JustinNusca – if there's issues getting clarity here I think it's fine to with just resolve the convo, rebase and merge, in the interests of getting this PR across the line. Can always push a hotfix later.

Reply:
Yeah, I'm finally circling back to this now that the relevant backend PR has been merged. 

There's a couple details that I don't think ever got sorted neatly (the designs have more-or-less settled on omitting the "Add Participant" button entirely, but the acceptance criteria still specifies it.

Everything is functional and polished enough that I agree with you: I'm going to proceed with merging, and if there _is_ outstanding details, handle it in a follow-up PR.